code-of-conduct.md

```diff
@@ -3,35 +3,35 @@
 
 We are a collective and we want to bring people in the community together to exchange ideas, make new friends, and build things as an inclusive, welcoming, and safe community. 
 
-Therefore, any harmful or discriminating behavior by ANYONE (contributors, hackers, mentors, volunteers, organizers, project leads, or absolutely anyone involved in our community) will NOT be tolerated and results in the offending person(s) being excluded from the group.
+Therefore, any harmful or discriminating behavior by ANYONE (volunteers, organizers, project leads, contributors, hackers, or absolutely anyone involved in our community) will NOT be tolerated and results in the offending person(s) being excluded from the group.
```

Comment:
perhaps just leave it with ANYONE and remove (volunteers, organizers, project leads, contributors, hackers, or absolutely anyone involved in our community) - kind of redundant and removing would simplify

Reply:
I'll change the order of the list to start with more volunteer/project focus. However, I would still like to call out the different roles to make sure folks understand that we don't put project leads, organizers or others above this rule. I think it helps to be clear that this is not only "anyone" but really "ANYONE" :)